system/extensions/esp32/wifi.toit

```diff
@@ -71,10 +80,15 @@ class WifiServiceDefinition extends NetworkServiceDefinitionBase:
     if not ssid or ssid.is_empty: throw "wifi ssid not provided"
     password/string := effective.get wifi.CONFIG_PASSWORD --if_absent=: ""
 
-    module ::= (state_.up: WifiModule.sta this ssid password) as WifiModule
-    if module.ap:
+    if module_ != null:
```

Comment:
This doesn't seem right. The NetworkState object is supposed to keep the network up as long as there are users.